modules/gmysqlbackend/schema.mysql.sql

```diff
@@ -3,13 +3,16 @@ CREATE TABLE domains (
   name                  VARCHAR(255) NOT NULL,
   master                VARCHAR(128) DEFAULT NULL,
   last_check            INT DEFAULT NULL,
-  type                  VARCHAR(6) NOT NULL,
+  type                  VARCHAR(8) NOT NULL,
   notified_serial       INT UNSIGNED DEFAULT NULL,
+  options               VARCHAR(64000) DEFAULT NULL,
```

Comment:
Is there a specific reason you are using VARCHAR here rather than TEXT? This way rows can potentially get very big, which can hamper InnoDB performance.